src/App.jsx

```diff
@@ -19,9 +21,14 @@ const App=()=> {
     <Header/>
     <Nav />
     <Routes>
-    <Route path="/" element={<Nav />} />
+   
+    <Route path="/" element={<Home />} />
+    <Route path="/home" element={<Home />} />
     <Route path="/articles" element={<Articles />} />
     <Route path="/articles/:article_id" element={<Body />} />
+    <Route path="/articles/:article_id/comments" element={<Comments />} />
+    <Route path="/articles/:article_id/comments/:comment_id" element={<CommentBody />} />
```

Comment:
Unless you plan to have a link to a specific comment and just show that comment on a page then I don't think you need this! Looks like you a rendering all the comments within article anyway which is what I would do :)